<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item rowspan="2" row="0" column="0" >
     <widget class="QTableWidget" name="llistaAlumnes" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text" >
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Foto</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QGraphicsView" name="graphicsView" >
      <property name="minimumSize" >
       <size>
        <width>226</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>317</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <spacer name="verticalSpacer" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>181</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>llistaAlumnes</zorder>
   <zorder>graphicsView</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFitxer" >
    <property name="title" >
     <string>Fitxer</string>
    </property>
    <addaction name="actionCarrega" />
    <addaction name="actionDesa" />
    <addaction name="separator" />
    <addaction name="actionSurt" />
   </widget>
   <addaction name="menuFitxer" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionCarrega" >
   <property name="text" >
    <string>Carrega...</string>
   </property>
  </action>
  <action name="actionDesa" >
   <property name="text" >
    <string>Desa</string>
   </property>
  </action>
  <action name="actionSurt" >
   <property name="text" >
    <string>Surt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
